### PR TITLE
Integrate lighthouse-ci

### PR DESCRIPTION
**Pull request by Eduardo-Morales-Alberti**
It could be a good contribution if the boilerplate had the tools to check the web performance.  
A good tool for it is lighthouse.  
Changes based on [GoogleChrome/lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/recipes)

### CODE REVIEW

File: lighthouse/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Dockerfile from https://github.com/GoogleChrome/lighthouse-ci/tree/main/docs/recipes/docker-client
+FROM node:12-buster-slim
```
Comment (omarlopesino) [inline lighthouse/Dockerfile:2]: Please publish this Dockerfile into the metadrop docker hub namespace, so this image can be easily usable in another projects that does not use the boilerplate.

Comment (Eduardo-Morales-Alberti) [reply lighthouse/Dockerfile:2]: @mistermoper I do not have permission to create a new repository on docker-hub, here is the repository https://github.com/Metadrop/docker-lighthouse-ci

**Comment by Eduardo-Morales-Alberti** [comment]
Update docker file https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/recipes/docker-client/Dockerfile

**Comment by Eduardo-Morales-Alberti** [comment]
Jenkins can not read the reports so it does not give any advantage over the already lighthouse chrome plugin.

**Pull request closed without merging**